src/Makefile.plugsched

```diff
@@ -22,5 +22,9 @@ collect:
 		CFLAGS_MODULE="$(GCC_PLUGIN_FLAGS)collect" $(vmlinux-deps)
 analyze:
 	python3 $(plugsched_tmpdir)/process.py ./vmlinux $(plugsched_tmpdir) $(plugsched_modpath)
-extract:
-	$(MAKE) CFLAGS_KERNEL="$(GCC_PLUGIN_FLAGS)extract" $(objs)
+
+extract: $(objs)
+
+%.extract:
```

Comment:
The target and prerequisite may be simplified. Refer to https://www.gnu.org/software/make/manual/html_node/Pattern-Rules.html

Reply:
_%.extract: %
          python3 $(plugsched_tmpdir)/extract.py $^ $(plugsched_tmpdir) $(plugsched_modpath)_

works for me.